src/MultipartSerializationWriter.cs

```diff
@@ -88,7 +88,7 @@ public void WriteByteArrayValue(string? key, byte[]? value)
     public void WriteDoubleValue(string? key, double? value) => throw new NotImplementedException();
     /// <inheritdoc/>
 #if NET5_0_OR_GREATER
-    public void WriteEnumValue<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)]T>(string? key, T? value) where T : struct, Enum => throw new NotImplementedException();
+    public void WriteEnumValue<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicFields)]T>(string? key, T? value) where T : struct, Enum => throw new NotImplementedException();
```

Comment:
Thanks for fixing this. Any chance you could also update `WriteCollectionOfEnumValues` on line 73 as well to align with the abstractions library too?

Reply:
Done, just needed to pull in the abstractions version bump.
I've also bumped the abstractions version to the one released this morning.